packages/geo/src/lib/catalog/shared/catalog.service.ts

```diff
@@ -200,4 +202,41 @@ export class CatalogService {
     }
   }
 
+  private retriveLayerInfoFormat(
+    layerNameFromCatalog: string,
+    layersQueryFormat: { layer: string, queryFormat: QueryFormat }[]): QueryFormat {
+
+    const currentLayerInfoFormat = layersQueryFormat.filter(f => f.layer === layerNameFromCatalog)[0];
```

Comment:
You can do .find instead of .filter if you want to return only one item instead of a list.